node/out_request.js

```diff
@@ -340,25 +434,22 @@ TChannelOutRequest.prototype.sendCallRequestContFrame = function sendCallRequest
 TChannelOutRequest.prototype.send = function send(arg1, arg2, arg3, callback) {
     var self = this;
 
-    var endpoint = String(arg1);
+    self.endpoint = String(arg1);
```

Comment:
Let’s all take note of this change. arg1 is coerced to string in the circuit health code as well. That’ll benefit from using this property instead. We may need to introduce the endpoint property to other request variants. cc @jcorbin